### ISSUE TITLE
Website links are dead

### ISSUE THREAD

**Issue by adampaigge** [sioodmy/dotfiles#81]
Hi Have been trygng to follow these dotfiles but all the links to the blog are dead, are there any other installation instructions

**Comment by sioodmy** [comment]
I deleted them because this repo is my personal configuration and not some kind of community framework. You probably shouldn't install it "as it is"

**Comment by sioodmy** [comment]
I might split my configs into separate flakes, so that everyone can try them out safely

**Comment by adampaigge** [comment]
Fair enough, functionally they are posted around as such but I am quite new to Nix. Grateful for the fact this is online at all

**Comment by sakshatshinde** [comment]
Would be great if there was a script to try some lighter version of the config

**Comment by sioodmy** [comment]
> Would be great if there was a script to try some lighter version of the config

I am working on something like that rn
https://github.com/sioodmy/andromeda
